onnx/test/helper_test.py

```diff
@@ -391,6 +391,30 @@ def test_make_tensor(self):  # type: () -> None
         )
         self.assertEqual(string_list, list(tensor.string_data))
 
+    def test_make_float16_tensor(self):  # type: () -> None
```

Comment:
can you add a test with raw=True for both FLOAT16 and BFLOAT16

Reply:
Added. To make it work, I made split_complex_to_pairs and numpy_array.view go into the else condition (raw=False). IIUC, the vals for raw=True should be the final binary data after conversion so no more conversion is needed inside this helper function (make_tensor).